src/main/ImGuiZepEditor.h

```diff
@@ -38,13 +49,36 @@ class ZepEditor_ImGui;
 class ImGuiZepEditor : public Zep::IZepComponent
 {
 public:
-	ImGuiZepEditor();
+	ImGuiZepEditor(std::string_view id);
 	virtual ~ImGuiZepEditor();
 
+	MQLIB_OBJECT static std::shared_ptr<ImGuiZepEditor> Create(std::string_view id);
```

Comment:
ahh, I see now -- you're passing a shared pointer across the boundary of a dll. Is this correct?

Reply:
Yes lots of dll shananigans.